x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/calculations/moving_average.tsx

```diff
@@ -174,7 +175,12 @@ const MovingAveragePopup = () => {
     <HelpPopover
       anchorPosition="upCenter"
       button={
-        <HelpPopoverButton onClick={() => setIsPopoverOpen(!isPopoverOpen)}>
+        <HelpPopoverButton
+          onClick={() => {
+            setIsPopoverOpen(!isPopoverOpen);
+            trackUiEvent('open_help_popover');
+          }}
+        >
```

Comment:
You're tracking every click, not just open clicks?

Also, is there a way this could be extracted into the `HelpPopoverButton` itself?